client/my-sites/comments/comment-list/index.jsx

```diff
@@ -132,18 +146,30 @@ export class CommentList extends Component {
 		} );
 	};
 
-	setCommentStatus = ( commentId, postId, status, options = { showNotice: true } ) => {
+	setCommentStatus = ( commentId, postId, status, options = { isUndo: false, persist: false, showNotice: true } ) => {
+		const { isUndo, persist, showNotice } = options;
```

Comment:
correspondingly to above, I often find that `doPersist` is a good name for an argument whose boolean intention is to mark something or not